components/sst_importer/src/service.rs

```diff
@@ -26,6 +26,6 @@ macro_rules! send_rpc_response {
                 $sink.fail(make_rpc_error(e))
             }
         };
-        res.map_err(|e| warn!("send rpc response"; "err" => %e))
+        let _ = res.map_err(|e| warn!("send rpc response"; "err" => %e)).compat().await;
```

Comment:
Should we do something with the result here rather than dropping it?

Reply:
We don't care about result and error type is `()`.